backend/logic/users/users.js

```diff
@@ -29,8 +30,15 @@ var insertStudentUser = async (username, password, email, id) => {
 
   try {
     await connection.beginTransaction();
-    await connection.query("INSERT INTO users VALUES(?, ?, ?);", [username, email, hash]);
-    await connection.query("INSERT INTO students VALUES(?, ?);", [id, username]);
+    await connection.query("INSERT INTO users VALUES(?, ?, ?);", [
```

Comment:
```suggestion
    await connection.query("INSERT INTO users (username, email, password) VALUES(?, ?, ?);", [
```

Reply:
Don't think it's necessary to have these if you fill in all columns. Also, didn't work on that method.

Reply:
Its more prevention in case we add or remove a column from this table in the future, it would break your code

Reply:
ok makes sense